lib/interface/cli/commands/project/create.cmd.js

```diff
@@ -48,18 +46,11 @@ const command = new Command({
         const {
             name: projectName,
             tag: tags,
-            variable: variables,
+            variable,
```

Comment:
not sure i like this change, let's try to find a different name for the variable you declare in line 53 instead